src/commonmark/en/content/developer/web-api.md

```diff
@@ -16464,6 +16476,236 @@ based on data element
+system, you will be working with the *eventFilters*
+resource. *POST* is used to create and *PUT* method is used to update. The event filter definitions are used in the
+Tracker Capture app to display relevant predefined "Working lists" in
+the tracker user interface.
+
+<table>
+<caption>Request Payload</caption>
+<colgroup>
+<col style="width: 33%" />
+<col style="width: 33%" />
+<col style="width: 33%" />
+</colgroup>
+<thead>
+<tr class="header">
+<th>Request Property</th>
+<th>Description</th>
+<th>Example</th>
+</tr>
+</thead>
+<tbody>
+<tr class="odd">
+<td>name</td>
+<td>Name of the filter.</td>
+<td>"name":"My working list"</td>
+</tr>
+<tr class="even">
+<td>description</td>
+<td>A description of the filter.</td>
+<td>"description":"for listing all events assigned to me".</td>
+</tr>
+<tr class="odd">
+<td>program</td>
+<td>Object containing the id of the program.</td>
+<td>"program" : {
+"id":"a3kGcGDCuk6"
+}</td>
+</tr>
+<tr class="even">
+<td>programStage</td>
+<td>Object containing the id of the program stage.</td>
+<td>"programStage" : {
+"id":"a3kGcGDCuk6"
+}</td>
+</tr>
+<tr class="odd">
+<td>organisationUnit</td>
+<td>Object containing the id of the organisation unit.</td>
+<td>"organisationUnit" : {
+"id":"a3kGcGDCuk6"
+}</td>
+</tr>
+<tr class="even">
+<td>eventQueryCriteria</td>
+<td>Object containing parameters for querying, sorting and filtering events.</td>
+<td>  
+  "eventQueryCriteria": {
+    "status": "COMPLETED",
+    "createdDate": {
+      "from": "2014-05-01",
+      "to": "2019-03-20"
+    },
+    "dataElements": ["a3kGcGDCuk6:EQ:1", "a3kGcGDCuk6"],
+    "filters": ["a3kGcGDCuk6:EQ:1"],
+    "programStatus": "ACTIVE",
+    "ouMode": "SELECTED",
+    "assignedUserMode": "PROVIDED",
+    "assignedUsers" : ["a3kGcGDCuk7", "a3kGcGDCuk8"],
+    "followUp": false,
+    "trackedEntityInstance": "a3kGcGDCuk6",
+    "events": ["a3kGcGDCuk7", "a3kGcGDCuk8"],
+    "fields": "eventDate,dueDate",
+    "order": "dueDate:asc,createdDate:desc"
+  }
+</td>
+</tr>
+</tbody>
+</table>
+
+<table>
+<caption>Event Query Criteria definition</caption>
+<colgroup>
+<col style="width: 33%" />
+<col style="width: 33%" />
+<col style="width: 33%" />
+</colgroup>
+<tbody>
+<tr class="odd">
+<td>followUp</td>
+<td>Used to filter events based on enrollment followUp flag. Possible values are true|false.</td>
+<td>"followUp": true</td>
+</tr>
+<tr class="even">
+<td>ouMode</td>
+<td>To specify the OU selection mode. Possible values are SELECTED| CHILDREN|DESCENDANTS|ACCESSIBLE|CAPTURE|ALL</td>
+<td>"ouMode": "SELECTED"</td>
+</tr>
+<tr class="odd">
+<td>assignedUserMode</td>
+<td>To specify the assigned user selection mode. Possible values are CURRENT| PROVIDED | NONE | ANY </td>
+<td>"assignedUserMode": "NONE"</td>
+</tr>
+<tr class="even">
+<td>assignedUsers</td>
+<td>To specify a list of assigned users for events.</td>
+<td>"assignedUsers": ["a3kGcGDCuk7", "a3kGcGDCuk8"]</td>
+</tr>
+<tr class="odd">
+<td>trackedEntityInstance</td>
```

Comment:
I dont think we have a usecase for this now. Perhaps we dont add it until we need it?

Reply:
Yep. Can remove it. Included it just because we had the parameter in our event listing API